Inseok_45/.env.sample

```
@@ -0,0 +1,12 @@
+ DATABASE_URL = "mysql://username:password@127.0.0.1:PORT/DATABASE_NAME"
+
+ DATABASE_URL = URL
```

```suggestion
DATABASE_URL="rdbmsType://username:password@iphost:portNumber/databaseName"
```
1. `DATABASE_URL`이라는 변수가 두번 들어가있습니다.
2. 해당 변수 또한 개인이 로컬 환경에서 어떠한 rdbms를 쓰는지 그 힌트가 될 수 있는 단어와, 포트번호를 지우고 중의적인 단어를 대신 사용해주세요.